lang/cs/Org.Apache.REEF.Driver/Bridge/ClrSystemHandlerWrapper.cs

```diff
@@ -165,12 +166,21 @@ public static void Call_ClrSystemCompletedEvaluator_OnNext(ulong handle, IComple
 
         public static void Call_ClrSystemHttpServer_OnNext(ulong handle, IHttpServerBridgeClr2Java clr2Java)
         {
-            using (LOGGER.LogFunction("ClrSystemHandlerWrapper::Call_ClrSystemHttpServer_OnNext"))
+            using (LOGGER.LogFunction("ClrSystemHandlerWrapper::Call_ClrSystemHttpServer_OnNext xxx"))
             {
-                GCHandle gc = GCHandle.FromIntPtr((IntPtr)handle);
-                ClrSystemHandler<IHttpMessage> obj = (ClrSystemHandler<IHttpMessage>)gc.Target;
-                obj.OnNext(new HttpMessage(clr2Java));
-            }      
+                try
+                {
+                    GCHandle gc = GCHandle.FromIntPtr((IntPtr) handle);
+                    ClrSystemHandler<IHttpMessage> obj = (ClrSystemHandler<IHttpMessage>) gc.Target;
+                    obj.OnNext(new HttpMessage(clr2Java));
+                }
+                catch (Exception ex)
+                {
+                    var msg = "Caught exception: " + ex.Message + ex.StackTrace;
+                    Console.WriteLine("WARNING: " + msg);
```

Comment:
I did not change that code. Not sure why I does show up here. I'll remove the `Console` stuff, though.